src/addons/phpmailer/lib/cronjob.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+ * @package redaxo\backup
+ */
+class rex_cronjob_phpmailer extends rex_cronjob
```

Comment:
brauch man speziell für den phpmailer oder brauch man das eigentlich allgemein für jegliche logfiles?

Reply:
da ich die Struktur anderer Logfiles nicht kenne - hier ist jede Mail eine Datei in einem Unter-Unterverzeichnis. Kann man aber gerne adaptieren.

Datenhygiene in REDAXO würde auch bedeuten, nicht nur Filesystem-Logs zu entfernen, sondern auch Logs in der Datenbank (bspw. History). Der Backup-Cronjob regelt das aktuell auch für sich.

Wenn's allgemein geht, gerne verbessern.

Reply:
diese Logs vom PHPMailer sind ein E-Mail-Archiv in Unterordnern und keine Logdatei / Protokoll. Hier werden sämtliche ausgehende Mails gespeichert. Daher ist es mehr oder weniger ein Spezialfall.